MCA/layouts/default.vue

```diff
@@ -26,65 +26,94 @@
         />
         
         <the-footer class="mcaayim__footer">
-            <a 
-                class="footer-nav__brand-name" 
-                :class="`footer-nav__brand-name--${viewTheme}`"
-                href="https://corsace.io"
-            >
-                <img
-                    class="corsace__icon"
```

Comment:
remove unused css assets like this one since we aren't gonna be going back to previous anyway

Reply:
removed the css class